pi/setup_pi.sh

```diff
@@ -96,8 +96,9 @@ else
   chmod 700 /usr/local/brave
 
   # change the hostname according to the config
+  old_hostname=$(</etc/hostname)
   hosts_file=$(</etc/hosts)
-  hosts_file="${hosts_file//raspberrypi/$hostname}"
+  hosts_file="${hosts_file//$old_hostname/$hostname}"
```

Comment:
Hmmm, actually, one more question about this one. Is `raspberrypi` already in the `/etc/hostname` file initially?

Reply:
Just found this article, which makes it looks like yes, it is in `/etc/hostname` initially.
https://www.tomshardware.com/how-to/raspberry-pi-change-hostname